packages/core/contracts-ovm/insured-bridge/implementation/OVM_BridgeDepositBox.sol

```diff
@@ -76,6 +76,7 @@ contract OVM_BridgeDepositBox is OVM_CrossDomainEnabled, OVM_Testable {
     event SetMinimumBridgingDelay(uint64 newMinimumBridgingDelay);
     event WhitelistToken(address l1Token, address l2Token, uint64 lastBridgeTime, address bridgePool);
     event DepositsEnabled(address l2Token, bool depositsEnabled);
+    // TODO: change the order of these to match the way they are used in the bridge pool.
     event FundsDeposited(
```

Comment:
is it worth putting all the uint64's together at the top of the event? We should keep them together though so that Solidity can try to optimize space.

Reply:
ye, this is a good idea. we should leave it until later though and go through looking for these kinds of optimisations.